### DISCUSSION TITLE
How to get the AzureCosmosDataProvider

### DISCUSSION THREAD

**Discussion by wouter-b** [thepirat000/Audit.NET#587] [Q&A]
Looking at the documentation I see the use of azureCosmosDataProvider, how do I get the AzureCosmosDataProvider? From DI?

IQueryable<AuditEvent> query = azureCosmosDataProvider.QueryEvents()
	.Where(ev => ev.Environment.MachineName == "HP")
	.OrderByDescending(ev => ev.Duration)
	.Take(10);

**Comment by thepirat000** [comment]
Please add more details. What are you auditing? 
Where do you want to get the data provider?
